utils/clean-stage1.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+[ -z "$CONFIG" ] && CONFIG="$PWD/config"
+[ -z "$CONFIG" ] && CONFIG=/src/config
+if [ ! -e "$CONFIG" ] ; then
+	echo "error: $CONFIG not found"
+	exit 1
+fi
+. "$CONFIG"
+
+[ -z "$butch_do_overwrite_on_copy" ] && butch_do_overwrite_on_copy=false
+[ -z "$butch_staging_dir" ] && butch_staging_dir=/opt
+
+relocate() {
```

Comment:
hmm... couldn't we just call the butch scripts from "$K" ?

Reply:
For relocate? Probably. It doesn't touch butch.db, nor does `butch-unlink`. We still need to provide our own `butch-rm` just to have control over how butch.db is edited.